util/start_test

```diff
@@ -90,7 +90,7 @@ def run_tests(tests):
 
     # print out flags
     logger.write('[tests: "{0}"]'.format(" ".join(files)))
-    if args.recurse:
+    if args.recurse and not os.path.isfile("NORECURSE"):
```

Comment:
Ah, could you try creating a `pyzmq-venv.notest`? I think this should accomplish what you're trying to do.

For `start_test` rule-files (where `notest` is a *rule*), `filename.rule` will act recursively, whereas `RULE` will act on the current directory only. 

note-to-self: This is should be documented in [TestSystem.rst](https://github.com/chapel-lang/chapel/blob/master/doc/rst/developer/bestPractices/TestSystem.rst)...